package.json

```diff
@@ -1,15 +1,18 @@
 {
   "name": "teamsfx",
   "scripts": {
-    "setup": "npm install && npm run bootstrap --hoist && npm run install-eslint",
+    "setup": "npm install && npm run build",
```

Comment:
this should be 
```json
{
  "postinstall": "npm run build"
}
```

Reply:
thank you!
if postinstall do the build job, so where to do the bootstrap, since lerna should help link each package.